src/assets/main.js

```diff
@@ -1,10 +1,6 @@
-let baseUrl = window.location.pathname;
-let fileIndex = baseUrl.indexOf("index.html");
-if (fileIndex > 0) {
-    baseUrl = baseUrl.slice(0, fileIndex);
-}
+let baseUrl = window.location.origin;
```

Comment:
origin returns only protocol, server + port, right? does this prevent us from serving from some deeper url (e.g. https://server:port/some/magic/url/test-editor-web/...)? Do we ever have the need for that? I don't know ...